tsl/test/sql/continuous_aggs_drop_chunks.sql

```diff
@@ -51,3 +51,67 @@ CALL refresh_continuous_aggregate('records_monthly', NULL, NULL);
 
 \set VERBOSITY default
 SELECT drop_chunks('records', '2000-03-16'::timestamptz);
+
+\set VERBOSITY terse
+DROP MATERIALIZED VIEW records_monthly;
+DROP TABLE records;
+DROP TABLE clients;
+\set VERBOSITY default
+
+-- Test how caggs are affected by drop_chunk due to refresh on drop.
+-- Demonstrates fix of issue #2592.
+
+CREATE OR REPLACE FUNCTION test_int_now() returns INT LANGUAGE SQL STABLE as 
+    $$ SELECT 125 $$;
+
+CREATE TABLE conditions(time_int INT NOT NULL, device INT, value FLOAT);
+SELECT create_hypertable('conditions', 'time_int', chunk_time_interval => 10);
+
+INSERT INTO conditions
+SELECT time_val, time_val % 4, 3.14 FROM generate_series(0,100,1) AS time_val;
+
+SELECT set_integer_now_func('conditions', 'test_int_now');
+CREATE MATERIALIZED VIEW conditions_7
+    WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE)
+    AS
+        SELECT time_bucket(7, time_int) as bucket,
+            SUM(value), COUNT(value)
+        FROM conditions GROUP BY bucket WITH DATA;
+SELECT materialization_hypertable_schema||'.'||materialization_hypertable_name AS mat_hyper
```

Comment:
I don't think `mat_hyper` is ever used. Is this query used at all?

Reply:
Thank you! It is left over after I queries the materialized table in the view, but it was redundant after I improved tests to cover corner cases better. I will remove this.